joomla/joomla-cms
# [5.3] [bugfix] Use iife for the scripts core and load validate as module

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/43714 .

### Summary of Changes

Use iife instead of esm for the script core.js and load the script validate.js with a `type=module`


### Testing Instructions

- Apply the PR
- run `npm install`
- Check that authenticating in the administrator works
- Create a new article, don't add anything and press save, the validation error should appear

### Actual result BEFORE applying this Pull Request

Leaking variables in the global scope

### Expected result AFTER applying this Pull Request

NO leaking variables in the global scope

### Link to documentations
Please select:


@Fedik
